packages/core/src/tracing/cls.js

```diff
@@ -149,6 +152,14 @@ class InstanaSpan {
       writable: true,
       enumerable: false
     });
+    // Indicates whether downstream suppression should be propagated.
```

Comment:
We need a bit more context here I think. And maybe some more thinking.

So any instantiation of `InstanaSpan` outputs

> instanaSpan.shouldSuppressDownstream = true

But right now the `InstanaSpan` does not suppress downstream, right?